components/compositing/compositor.rs

```diff
@@ -825,6 +825,11 @@ impl<Window: WindowMethods> IOCompositor<Window> {
                     warn!("Sending reload to constellation failed ({}).", e);
                 }
             }
+
+            WindowEvent::ToggleProfiler => {
```

Comment:
Can you rename this to ToggleWebRenderProfiler? Because their is also a regular profiler.